{
  "Class": "vulnerability",
  "ID": "oval:com.redhat.cve:def:202441989",
  "Version": "636",
  "Metadata": {
    "Title": "python-django: Memory exhaustion in django.utils.numberformat.floatformat() (moderate)",
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2024-41989",
        "RefURL": "https://access.redhat.com/security/cve/CVE-2024-41989"
      }
    ],
    "Description": "DOCUMENTATION: A security issue was found in Django. If 'floatformat' received a string representation of a number in scientific notation with a large exponent, it could lead to significant memory consumption. To avoid this, decimals with more than 200 digits are now returned as is. \n            STATEMENT: This issue is categorized as moderate severity rather than important because, while it has the potential to cause significant memory consumption under specific conditions, the likelihood of such a scenario occurring in typical applications is relatively low. The issue arises primarily when processing extremely large numbers in scientific notation, which is not a common use case in most Django applications. Additionally, the impact is localized to the floatformat function, and the issue does not expose sensitive data or compromise the integrity of the application.\n            MITIGATION: Mitigation for this issue is either not available or the currently available options do not meet the Red Hat Product Security criteria comprising ease of use and deployment, applicability to widespread installation base or stability.",
    "Advisory": {
      "From": "secalert@redhat.com",
      "Severity": "Moderate",
      "Issued": {},
      "Updated": {
        "Date": "2024-08-29"
      },
      "Cves": [
        {
          "CveID": "CVE-2024-41989",
          "Cvss3": "7.5/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
          "Cwe": "CWE-400",
          "Impact": "moderate",
          "Href": "https://access.redhat.com/security/cve/CVE-2024-41989",
          "Public": "20240806:1300"
        }
      ],
      "AffectedCpeList": [
        "cpe:/a:redhat:satellite:6.10",
        "cpe:/a:redhat:satellite:6.10::el7",
        "cpe:/a:redhat:satellite_capsule:6.10",
        "cpe:/a:redhat:satellite_capsule:6.10::el7"
      ],
      "Affected": {
        "Resolution": {
          "State": "Affected"
        }
      }
    }
  },
  "Criteria": {
    "Operator": "OR",
    "Criterions": [
      {
        "Comment": "Red Hat Enterprise Linux must be installed",
        "TestRef": "oval:com.redhat.cve:tst:20134120010"
      }
    ],
    "Criterias": [
      {
        "Operator": "AND",
        "Criterions": [
          {
            "Comment": "Red Hat Enterprise Linux 7 is installed",
            "TestRef": "oval:com.redhat.cve:tst:20134120009"
          }
        ],
        "Criterias": [
          {
            "Operator": "OR",
            "Criterias": [
              {
                "Operator": "AND",
                "Criterions": [
                  {
                    "Comment": "python-django is installed",
                    "TestRef": "oval:com.redhat.cve:tst:201919844001"
                  },
                  {
                    "Comment": "python-django is signed with Red Hat redhatrelease2 key",
                    "TestRef": "oval:com.redhat.cve:tst:201919844002"
                  }
                ]
              },
              {
                "Operator": "AND",
                "Criterions": [
                  {
                    "Comment": "python2-django is installed",
                    "TestRef": "oval:com.redhat.cve:tst:201919844003"
                  },
                  {
                    "Comment": "python2-django is signed with Red Hat redhatrelease2 key",
                    "TestRef": "oval:com.redhat.cve:tst:201919844004"
                  }
                ]
              }
            ]
          }
        ]
      }
    ]
  }
}